src/main/java/org/opentripplanner/updater/transportation_network_company/NoApiTransportationNetworkCompanyDataSource.java

```diff
@@ -0,0 +1,89 @@
+package org.opentripplanner.updater.transportation_network_company;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.opentripplanner.routing.transportation_network_company.ArrivalTime;
+import org.opentripplanner.routing.transportation_network_company.RideEstimate;
+import org.opentripplanner.routing.transportation_network_company.TransportationNetworkCompany;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This data source is to model a transportation network company for which no API exists to calculate real-time arrival
+ * estimates or ride estimates. The config for this updater can include a default value for the estimated arrival time
+ * which will always be provided when estimates for the arrival time are desired.
+ */
+public class NoApiTransportationNetworkCompanyDataSource extends TransportationNetworkCompanyDataSource {
+    private static final Logger LOG = LoggerFactory.getLogger(NoApiTransportationNetworkCompanyDataSource.class);
+
+    /**
+     * The default arrival time in seconds to respond with for all valid arrival time estimate requests. Defaults to 0.
+     */
+    private int defaultArrivalTimeSeconds = 0;
+
+    /**
+     * Whether or not the TNC service being modeled is wheelchair accessible. Defaults to false.
+     */
+    private boolean isWheelChairAccessible = false;
+
+    /**
+     * Configures the data source. At this point, the only values the config could contain are the following:
+     * - defaultArrivalTimeSeconds
+     * - isWheelChairAccessible
+     */
+    public NoApiTransportationNetworkCompanyDataSource(JsonNode config) {
+        defaultArrivalTimeSeconds = config.path("defaultArrivalTimeSeconds").asInt();
+        isWheelChairAccessible = config.path("isWheelChairAccessible").asBoolean();
+    }
+
+    /**
+     * For testing purposes only.
+     */
+    public NoApiTransportationNetworkCompanyDataSource() {
+        defaultArrivalTimeSeconds = 123;
+        isWheelChairAccessible = true;
+    }
```

Comment:
Instead of a simple no-arg constructor, it might be more clear to use a static method `NoApiTransportationNetworkCompanyDataSource .createTestDataSource` to generate this test instance.

Reply:
I refactored this so that it accepts arguments to be consistent with other TNC test cases.